testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-gcs/tempto-configuration.yaml

```diff
@@ -0,0 +1,19 @@
+databases:
+    hive:
+        host: hadoop-master
+        jdbc_driver_class: org.apache.hive.jdbc.HiveDriver
+        jdbc_url: jdbc:hive2://${databases.hive.host}:10000
+        jdbc_user: hive
+        jdbc_password: na
+        jdbc_pooling: false
+        schema: default
+        prepare_statement:
+            - USE ${databases.hive.schema}
```

Comment:
Question unrelated to this PR - can we use this to for example run `CREATE SCHEMA` against `trino` runner?

e.g. File HMS doesn't allow creating views in non-existent schemas so I have some PTs where the test itself runs CREATE SCHEMA IF NOT EXISTS and I'd maybe want the schema creation to happen here.